mmr/mmr.go

```diff
@@ -131,7 +147,11 @@ func (m *MMR) bagRHSPeaks(rhsPeaks [][]byte) []byte {
 		if lp, rhsPeaks = pop(rhsPeaks); lp == nil {
 			panic("pop")
```

Comment:
@oshorefueled Please remove every `panic` in the code body, errors should be handled.